cortex-mixin/config.libsonnet

```
@@ -37,6 +37,7 @@
       store_gateway: '(store-gateway|cortex$)',
       gateway: '(gateway|cortex-gw|cortex-gw-internal)',
       compactor: 'compactor.*',  // Match also custom compactor deployments.
+      alertmanager: '(alertmanager.*|cortex$)',
```

I would just use "alertmanager" as default. The alertmanager is not running as part of the single binary (at least not by default) so I would avoid having `|cortex`. I would also avoid `alertmanager.*` by default since it could match other unrelated alertmanager installations (eg. Prometheus's vanilla one).

```suggestion
      alertmanager: 'alertmanager',
```